.codeclimate.yml

```diff
@@ -1,7 +1,7 @@
 ---
 engines:
   bundler-audit:
-    enabled: true
+    enabled: false
```

Comment:
Because there's no `Gemfile.lock` and thus `bundler-audit` doesn't work. But check out the work on #77 - maybe that isn't necessary